schedule/yast/modify_existing_partition_sle.yaml

```diff
@@ -42,7 +42,7 @@ test_data:
 
 conditional_schedule:
   reconnect_mgmt_console:
-    BACKEND:
+    ARCH:
```

Comment:
Now you can remove comments in line 25 and 27. They don't provide any additional info.

Reply:
They were there before. Let's merge to fix the failure and address comments later on.